.github/workflows/tics.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
     - main
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Would you also want `cancel-in-progress: true`?

Reply:
I thought no, because we want results for each merge into `main`, and then I'm not sure how well does TICS behave if interrupted.

Reply:
Or rather, we have evidence that it doesn't behave well if interrupted…

https://github.com/canonical/mir/actions/runs/10559902271